ops/lib.rs

```diff
@@ -398,6 +427,8 @@ fn is_handle_scope(arg: &syn::FnArg) -> bool {
     || tokens(arg).ends_with(": & mut v8 :: HandleScope < 'a >")
     || tokens(arg).ends_with(": & mut deno_core :: v8 :: HandleScope")
     || tokens(arg).ends_with(": & mut deno_core :: v8 :: HandleScope < 'a >")
+    || tokens(arg).contains("mut v8 :: HandleScope")
+    || tokens(arg).ends_with(": & mut v8 :: HandeScope < 'scope >")
```

Comment:
This case and the first two are covered by the `.contains("mut v8 :: HandleScope")` above, but I don't know if that should be there.

Should we just use regex in this function for the lifetime parameter?